backend/src/main/java/ai/verta/modeldb/versioning/blob/container/EnvironmentContainer.java

```diff
@@ -208,7 +208,8 @@ private void validateEnvironmentVariableName(String name) throws ModelDBExceptio
   private String computeSHA(EnvironmentBlob blob) throws NoSuchAlgorithmException {
     StringBuilder sb = new StringBuilder();
     sb.append("env:");
-    for (EnvironmentVariablesBlob environmentVariablesBlob : blob.getEnvironmentVariablesList()) {
+    for (EnvironmentVariablesBlob environmentVariablesBlob :
+        new HashSet<>(blob.getEnvironmentVariablesList())) {
```

Comment:
Why this change required? it just for loop you can use whatever I mean List or Set I think it doesn't affect to hash also.

Reply:
HashSet ignores order of elements. That's why I have used it. But it doesn't guarantee order, so I better change it to sort.